lib/index.js

```diff
@@ -0,0 +1,108 @@
+/*
+ * Bedrock Agreement Module.
+ *
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+/* jshint node:true */
+'use strict';
+
+var async = require('async');
+var bedrock = require('bedrock');
+var brPermission = require('bedrock-permission');
+var database = require('bedrock-mongodb');
+var eventLog = require('bedrock-event-log').log;
+var BedrockError = bedrock.util.BedrockError;
+var uuid = require('uuid').v4;
+
+// load config defaults
+require('./config');
+
+bedrock.events.on('bedrock.test.configure', function() {
+  require('./test.config');
+});
+
+// module permissions
+var PERMISSIONS = bedrock.config.permission.permissions;
+
+// module API
+var api = {};
+module.exports = api;
+
+var logger = bedrock.loggers.get('app');
+
+/**
+ * Add an agreement event
+ *
+ * @param actor the Identity performing the action.
+ * @param agreements a string or array of strings referencing agreements
+ * @param callback(err, result) called once the operation completes
+ */
+api.add = function(actor, agreements, callback) {
```

Comment:
I think we actually want this to be "api.accept" given all of the other changes and discussions we've had. And the permission can be `AGREEMENT_ACCEPT`.